instrumentation/basic_logger.go

```diff
@@ -86,6 +86,29 @@ func Stringable(key string, value fmt.Stringer) *Field {
 	return &Field{Key: key, String: value.String(), Type: StringType}
 }
 
+func StringableSlice(key string, values interface{}) *Field {
+	builder := strings.Builder{}
+	builder.WriteString("[")
+
+	switch reflect.TypeOf(values).Kind() {
```

Comment:
this is a performance hit. We need to think if we can defer execution of this function, so that in production we can choose "debug levels" or whatnot, thereby saving on the performance hit, or at least executing the expensive logging-related stuff in a background goroutine. I suggest to open an issue on that and make sure to decide if we want to handle it before V1 goes live.